### PR TITLE
Clean commit of extracting totals from mtlist budgets.

### PR DESCRIPTION
**Pull request by briochh**
In and out totals from the sw and gw budget blocks of the mt3d
list file using `MtListBudget().parse()`.

Adding support for when mt3d kper > 99999 when parsing list file.

Also now extracting in-out and discrepancy (percent) from the gw budget blocks.

Added test for selecting diff=False when parsing mt listfile.

TODO: in-out and discrepancy (percent) from the sw budget blocks

### CODE REVIEW

**Comment by langevin-usgs** [comment]
@briochh, it looks like you've made quite a few changes here, and they look like good changes, but I haven't tested it.  Would you say this PR is ready to be accepted?  I see there is a TODO in the code.   Is that something you were planning to address now?

**Comment by briochh** [comment]
Hi @langevin-usgs, It works as is (I have been using it for a while, for mt3d-usgs list files at least) but yes, not quite complete as the surface water components of the list file aren’t quite handled fully yet. I see you guys are planning for another release shortly, I can try getting that remaining TODO plumbed in before then?

**Comment by briochh** [comment]
Hi @langevin-usgs, this should be good now (pending Travis). It's a bit hacky, caluculating the SW diff (relies on the names outputed by mt3d), happy to here suggestions for how to do it better!

File: flopy/utils/mtlistfile.py [hunk 1]
```diff
@@ -68,8 +73,9 @@ def parse(self, forgive=True, diff=True, start_datetime=None,
         Returns
         -------
         df_gw,df_sw : pandas.DataFrame
-            a dataframe for the groundwater mass and (optionally) surface-water mass budget.
-            if the SFT process is not used, only one dataframe is returned.
+            a dataframe for the groundwater mass and
+            (optionally) surface-water mass budget.
+            If the SFT process is not used, only one dataframe is returned.
```
Comment (jdhughes-usgs) [inline flopy/utils/mtlistfile.py]: would it be better to indicate df_sw is None if the SFT process is not used

**Pull request merged**